test/e2e/testdata/bad-csv/example-operator.v0.1.0.yaml

```diff
@@ -0,0 +1,25 @@
+---
+schema: olm.package
+name: packageA
+defaultChannel: stable
+---
+schema: olm.channel
+package: packageA
+name: stable
+entries:
+  - name: example-operator.v0.1.0
+---
+schema: olm.bundle
+name: example-operator.v0.1.0
+package: packageA
+image: quay.io/olmtest/example-operator-bundle:0.1.0
+properties:
+  - type: olm.gvk
+    value:
+      group: example.com
+      kind: TestA
+      version: v1alpha1
+  - type: olm.package
+    value:
+      packageName: packageA
+      version: 1.0.0
```

Comment:
Same comment as [below](https://github.com/operator-framework/operator-lifecycle-manager/pull/2673/files#r861210613) - are we using this manifest anywhere, or can we drop it from the set of changes?

Reply:
same comment as below.